opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -636,7 +650,8 @@ namespace Opm
     {
         const RestartConfig& restartConfig = eclipseState_->getRestartConfig();
         const int reportStepNum = timer.reportStepNum();
-        std::vector<data::CellData> cellData = detail::getCellData( phaseUsage_, physicalModel, restartConfig, reportStepNum );
+        std::vector<data::CellData> cellData = detail::getCellData( phaseUsage_, physicalModel, restartConfig, reportStepNum,
+                                                                    output_ && parallelOutput_->isIORank() );
```

Comment:
This is an important change I think. I guess all data will be gathered on rank 0 and written only from there? Will not this result in incomplete data being stored, since we should extract data from all ranks, not just from rank 0?